dokku

```diff
@@ -58,6 +58,8 @@ fi
 
 if [[ $(id -un) != "root" && $1 =~ ^plugin:.* && $1 != "plugin:help" && $1 != "plugin:list" ]] || [[ $(id -un) != "root" && $1 == "ssh-keys:add" ]]; then
   dokku_log_fail "This command must be run as root"
+else
+  export SSH_USER=root
```

Comment:
Doesn't this force-override it always? We should only set the SSH_USER here if its one of the commands that requires root.